tailwind.config.js

```diff
@@ -44,10 +44,10 @@ module.exports = {
       scale: {
         '-100': '-1',
       },
-      colors: {
-        'press-indigo': '#003D69',
-        'press-grass': '#00593E',
-      },
+    },
+    colors: {
+      'press-indigo': '#003D69',
+      'press-grass': '#00593E',
```

Comment:
FYI: I stand corrected on this. I must've tested it incorrectly. The way @jenbreese had this configured originally before I made these changes in PR #5 was correct! 😃

Reply:
How about adding "Magenta" and "Steel" while we're here?

Reply:
I was able to locate the new colors `press-sand` and `press-bay-light-light`. Do you know where I can find the Magenta and Steel hex codes? 
![image](https://github.com/SU-SWS/press-nextjs/assets/34019925/d0db9b1f-478f-4c3f-b1e9-6f136099ad91)

Reply:
No idea. We'll come back to it